ILRuntime/Runtime/Intepreter/ILIntepreter.cs

```diff
@@ -1948,7 +1950,7 @@ public object Run(ILMethod method, object instance, object[] p)
                                             CLRType t = type as CLRType;
                                             int idx = (int)ip->TokenLong;
                                             var f = t.GetField(idx);
```

Comment:
Used to determine below if it is a boxed value (by checking `FieldType == typeof(object)`)